### ISSUE TITLE
Zenroom build fails on arm pc

### ISSUE THREAD

**Issue by matteo-cristino** [dyne/Zenroom#806]
Using 
```
make meson
```
the following error are reported
```
[1/14] Compiling C object libzenroom.dylib.p/.._src_p256-m.c.o
FAILED: libzenroom.dylib.p/.._src_p256-m.c.o 
gcc -Ilibzenroom.dylib.p -I. -I../build -I../src -I../lib/milagro-crypto-c/include -Imilagro-crypto-c/include -I../build/milagro-crypto-c/include -I../lib/ed25519-donna -I../lib/lua54/src -I../lib/mimalloc/include -fdiagnostics-color=always -Wall -Winvalid-pch -Wextra -std=c99 -O3 -g '-DVERSION="v4.12.0"' '-DCURRENT_YEAR="2024"' '-DCOMMIT="f9f2d8ca"' -D_POSIX_C_SOURCE=200112L -D_DARWIN_C_SOURCE -DARCH_OSX -O2 -fstack-protector-all -D_FORTIFY_SOURCE=2 -fno-strict-overflow -fPIC -MD -MQ libzenroom.dylib.p/.._src_p256-m.c.o -MF libzenroom.dylib.p/.._src_p256-m.c.o.d -o libzenroom.dylib.p/.._src_p256-m.c.o -c ../src/p256-m.c
../src/p256-m.c:247:12: error: invalid output constraint '+l' in asm
            : [x] "+l"(x), [y] "+l"(y),
                  ^
../src/p256-m.c:263:12: error: invalid output constraint '+l' in asm
            : [x] "+l"(x), [y] "+l"(y), [u] "=&l"(tmp1)
                  ^
2 errors generated.
[9/14] Compiling C object libzenroom.dylib.p/.._src_zen_octet.c.o
ninja: build stopped: subcommand failed.
make: *** [linux-meson] Error 1
```

**Comment by matteo-cristino** [comment]
Looking at GCC documentation https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html the correct flag to use when using asm should be `__GCC_ASM_FLAG_OUTPUTS__`

**Comment by matteo-cristino** [comment]
solved in #808
